### PR TITLE
Rov 162 account s home screen UI

### PR DESCRIPTION
**Pull request by Varuno8**

### CODE REVIEW

File: src/app/application/account-details/account-details.component.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
+.account-name {
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.css:1]: use the standard css from the style.css

File: src/app/application/account-details/account-details.component.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+}
+.box-title {
+  color: #18b091;
+  margin-top: 16px;
+  margin-left: 22px;
+  font-family: "Roboto";
+  font-size: 20px;
+  font-weight: 500;
+}
+.secondary-text {
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.css:24]: use `.text-secondary` class for this

File: src/app/application/account-details/account-details.component.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  color: #212529;
+  display: block;
+  margin-left: 57px;
+  margin-top: 35px;
+  font-family: "Roboto";
+  font-size: 16px;
+  font-weight: 400;
+}
+.secondary-textmuted {
+  color: #6c757d;
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.css:34]: use the existing css class for this

File: src/app/application/account-details/account-details.component.html [hunk 4]
```diff
@@ -1,14 +1,41 @@
...
-        </div> 
-        <div class="col-3 text-end">
-            <button class="btn btn-primary btn-lg" (click)="openUploadModal()">
-                <i class="bi bi-plus"></i>
-                Upload Statement
-            </button>
-        </div>
+  <div class="row mx-4 m-4">
+    <div class="col-9">
+      <h1 class="account-name">{{ accountName }}</h1>
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.html:4]: this text should be in-line with the upload button

File: src/app/application/account-details/account-details.component.css [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  margin-left: 72px;
+}
+.box {
+  width: 199px;
+  height: 140px;
+  margin-top: 38px;
+  margin-left: 72px;
+  border: 1px solid rgba(26, 188, 155, 1);
+}
+.box-title {
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.css:16]: use the css class from the existing bootstrap one's or style.css

File: src/app/application/account-details/account-details.component.html [hunk 6]
```diff
@@ -1,14 +1,41 @@
...
+      </button>
+    </div>
+  </div>
 
-</div>
\ No newline at end of file
+  <div class="row m-4">
+    <div class="col-3">
+      <div class="box">
+        <h5 class="box-title">Opening Balance</h5>
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.html:17]: this text should be center aligned

File: src/app/application/account-details/account-details.component.html [hunk 7]
```diff
@@ -1,14 +1,41 @@
...
+        <span class="secondary-text">₹6654.00</span>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="box">
+        <h5 class="box-title">Credit Amount</h5>
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.html:23]: this should be center-aligned

File: src/app/application/account-details/account-details.component.html [hunk 8]
```diff
@@ -1,14 +1,41 @@
...
+        <span class="secondary-text">₹4314.00</span>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="box">
+        <h5 class="box-title debit-text">Debit Amount</h5>
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.html:29]: this should be center-aligned

File: src/app/application/account-details/account-details.component.html [hunk 9]
```diff
@@ -1,14 +1,41 @@
...
+        <span class="secondary-text">₹5582.00</span>
+      </div>
+    </div>
+    <div class="col-3">
+      <div class="box">
+        <h5 class="box-title">Closing Balance</h5>
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.html:35]: this should be center-aligned

File: src/app/application/account-details/account-details.component.ts [hunk 10]
```diff
@@ -34,17 +36,19 @@ export class AccountDetailsComponent implements OnInit, OnDestroy {
...
   getAccountDetails() {
-    this.subscription.add(this.accountDetailsService.getAccountDetails(this.accountId).subscribe((result: AccountDetail) => {
-      console.log(result);
-      this.accountInfo = result;
-    }));
+    this.subscription.add(
+      this.accountDetailsService.getAccountDetails(this.accountId).subscribe((result: AccountDetail) => {
+        console.log(result);
+        this.accountInfo = result;
+        this.accountName = result.AccountName; // Update accountName with the retrieved account name
```
Comment (rachittyagi) [inline src/app/application/account-details/account-details.component.ts:46]: why we are holding this in a separate variable?

**Review by rachittyagi — CHANGES_REQUESTED** [review]

File: src/app/application/account-details/account-details.component.html (hunk 9 above)
Comment (Varuno8) [reply src/app/application/account-details/account-details.component.html:35]: resolved

File: src/app/application/account-details/account-details.component.css (hunk 1 above)
Comment (Varuno8) [reply src/app/application/account-details/account-details.component.css:1]: what is the style.css meaning?

File: src/app/application/account-details/account-details.component.css (hunk 3 above)
Comment (Varuno8) [reply src/app/application/account-details/account-details.component.css]: used the style.css for colour or font size but for customised margin I have to make these classes in acc-detail.comp.css file

**Review by rachittyagi — APPROVED** [review]

**Pull request merged**